pom.xml

```diff
@@ -47,7 +47,7 @@
     <jenkins.host.address />
     <slaveAgentPort />
     <java.level>8</java.level>
-    <jenkins.version>2.263.1</jenkins.version>
+    <jenkins.version>2.277.1</jenkins.version>
```

Comment:
I believe this is motivated by the `durable-task` bump below, but I don't think it is necessary.

Reply:
yes this is motivated by de durable task if we can "fix that" I have no need to bump up the Jenkins version. 
actually, I prefer to be compatible with the lowest version possible.